notebooks/poisoning_attack_witches_brew.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
Could you please update the notebooks/README with a link and abstract for this new notebook?